src/user_dto.py

```diff
@@ -70,4 +70,5 @@ def from_csv(cls, csv_str):
         for i in range(len(data)):
             if data[i].isdigit():
                 data[i] = int(data[i])
+
```

Comment:
Unnecessary newline

Reply:
literaly pycharm sees it as a problem, when there is no newline there XD

Reply:
Ok, in that case I retract my comment. I joust found it strange that the only change you made to this file was adding a newline, and I thought it was a mistake